docs/adr/0001-mvp-webidl.md

```diff
@@ -0,0 +1,179 @@
+# Build an MVP based on WebIDL and a manual workflow
+
+* Status: accepted
+* Deciders: rfkelly, linacambridge, eoger, jhugman, tarikeshaq
+* Date: 2020-07-01 (or thereabouts)
```

Comment:
This document sums up how I think we got to the current "push ahead with WebIDL to get something out the door" approach that we're executing on for our first version; here's a [convenient rendered view](https://github.com/mozilla/uniffi-rs/blob/docs-and-adrs/docs/adr/0001-mvp-webidl.md).

In particular, I've tried to emphasize here that we're deliberately making choices that are short-term focused, and that we may want to revisit in future. If something here doesn't ring true to you, please, let's discuss and refine this doc!